<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>306</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuSorting">
     <property name="title">
      <string>Sorting</string>
     </property>
     <addaction name="actionArtist"/>
     <addaction name="actionTitle"/>
     <addaction name="actionAlbum"/>
     <addaction name="actionGenre"/>
     <addaction name="separator"/>
     <addaction name="actionSort_ascending"/>
     <addaction name="actionSort_descending"/>
    </widget>
    <addaction name="menuSorting"/>
    <addaction name="actionDark_Theme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDark_Theme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Dark Theme</string>
   </property>
  </action>
  <action name="actionArtist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Artist</string>
   </property>
  </action>
  <action name="actionTitle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </action>
  <action name="actionAlbum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Album</string>
   </property>
  </action>
  <action name="actionGenre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </action>
  <action name="actionSort_ascending">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sort ascending</string>
   </property>
  </action>
  <action name="actionSort_descending">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sort descending</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
